src/core/scene/a-scene.js

```diff
@@ -50,10 +50,17 @@ module.exports.AScene = registerElement('a-scene', {
         this.isMobile = isMobile;
         this.isScene = true;
         this.object3D = new THREE.Scene();
+        this.renderUsedCamera = null;
+        var self = this;
+        this.object3D.onAfterRender = function (renderer, scene, camera) {
+          // THREE may swap the camera used for the rendering if in VR, so expose it
+          self.renderUsedCamera = camera;
```

Comment:
Should we pass the camera as a third argument of the tock call? It might be cleaner